### PR TITLE
Alex statuses table

### PR DESCRIPTION
**Pull request by whiteadam2**

### CODE REVIEW

File: apps/data-scanner/lib/db/migrations/sqls/20220402120235-add-statuses-table-up.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS statuses;
+CREATE TABLE statuses(
+	status_id SERIAL PRIMARY KEY,
+	full_name VARCHAR (50),
+	username VARCHAR (30),
+	was_online VARCHAR (30),
```
Comment (yzhbankov) [inline apps/data-scanner/lib/db/migrations/sqls/20220402120235-add-statuses-table-up.sql:6]: was_online and check_date should be a timestamp data type in other case usage of these table will be very hard

Comment (whiteadam2) [reply apps/data-scanner/lib/db/migrations/sqls/20220402120235-add-statuses-table-up.sql]: done

File: apps/data-scanner/lib/models/session/Sessions.mjs [hunk 2]
```diff
@@ -50,7 +51,18 @@ export class Sessions {
...
         for (const client of this.#pool) {
             const result = await client.invoke(command);
-            await writeToFile(result.users);
+            // await writeToFile(result.users);
+
+            for (let user of result.users) {
+                const params = {
+                    fullName: user.firstName + user.lastName,
+                    username: user.username,
+                    wasOnline: humanReadableDate(user.status.wasOnline),
```
[the remark this answers is not in the document]
Comment (whiteadam2) [reply apps/data-scanner/lib/models/session/Sessions.mjs]: done

File: shared/repository/lib/api/documents/StatusesRepository.mjs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+        super(db);
+    }
+
+    /**
+     * @method
+     * @param {number} statusId - status identifier
+     * @return {Promise<Object>} returns data saved in DB
+     */
+    async read({ statusId }) {
```
[the remark this answers is not in the document]
Comment (whiteadam2) [reply shared/repository/lib/api/documents/StatusesRepository.mjs]: done

File: apps/data-scanner/lib/models/statuses/Statuses.mjs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+     * @property save
+     * @property update
+     */
+
+    /**
+     * @method
+     * @param {Number} statusId - status identifier
+     * @returns {Promise<Object>}
+     */
+    async read({ statusId }) {
```
[the remark this answers is not in the document]
Comment (whiteadam2) [reply apps/data-scanner/lib/models/statuses/Statuses.mjs]: done

File: apps/data-scanner/lib/db/migrations/sqls/20220402120235-add-statuses-table-up.sql [hunk 5]
```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS statuses;
+CREATE TABLE statuses(
+	id SERIAL PRIMARY KEY,
+	phone_number VARCHAR(20) NOT NULL,
+	username TEXT DEFAULT NULL,
+	was_online TIMESTAMP DEFAULT NULL,
+	check_date TIMESTAMP NOT NULL
```
Comment (yzhbankov) [inline apps/data-scanner/lib/db/migrations/sqls/20220402120235-add-statuses-table-up.sql:7]: for the future, you can use ```TIMESTAMP NOT NULL DEFAULT NOW()```

**Review by yzhbankov — APPROVED** [review]

**Pull request merged**